CommonAssemblyInfo.cs

```diff
@@ -1,4 +1,4 @@
 using System.Reflection;
 
-[assembly: AssemblyVersion("6.5.0")]
-[assembly: AssemblyFileVersion("6.5.0")]
+[assembly: AssemblyVersion("6.90.0")]
```

Comment:
Before clicking create it should give you the option to switch the target branch.